Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -0,0 +1,172 @@
+  - type: SolutionContainerManager
+    solutions:
+      hypospray:
+        maxVol: 50
+  - type: RefillableSolution
+    solution: hypospray
+  - type: ExaminableSolution
+    solution: hypospray
+  - type: Hypospray
+    onlyAffectsMobs: false
+  - type: UseDelay
+    delay: 0.2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)